### PR TITLE
docs(examples): rename hello.ts to [...hello].ts

### PR DESCRIPTION
**Pull request by qin-guan**
### 🔗 Linked issue

https://github.com/nuxt/framework/issues/5209

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Fixes a bad server route in useFetch example

Resolves https://github.com/nuxt/framework/issues/5209

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]

**Comment by pi0** [comment]
Thanks for this PR @qin-guan 💚 It seems working an good fix. Since we are now using named route params, we can also replace `parseURL().pathname` ([here](https://github.com/nuxt/framework/blob/eb12c7a845f0744d90afc10cd15c38463633112c/examples/composables/use-fetch/server/api/hello.ts#L5)) to `event.context.params.id`.

**Comment by qin-guan** [comment]
Hey @pi0, sorry for the late update, 😔 PTAL

File: examples/composables/use-fetch/server/api/[...hello].ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { useQuery } from 'h3'
```
Comment (pi0) [inline examples/composables/use-fetch/server/api/[...hello].ts:1]: We don't need it. It is auto imported :)

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks for working on this!

**Comment by qin-guan** [comment]
Thanks for this first PR opportunity, and for all the work y'all are doing with Nuxt, truly amazing! 🙌🏻 🙏🏻

**Pull request merged**